Sprint-2/extend/format-time.js

```diff
@@ -22,3 +31,33 @@ console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
 );
+
+console.assert(
```

Comment:
This is great, you've written out a few varied 'test cases' here to check the code will work in all situations. This is a really good approach and when you start writing tests for your code, will work well for you.